src/unit/test_kvstore.c

```diff
@@ -0,0 +1,206 @@
+
+#include "../kvstore.c"
+#undef UNUSED
```

Comment:
Given the rest of this looks good, you could maybe add this to another PR. Maybe we can update the definition in test_help to be ```
#ifndef UNUSED
#define UNUSED
#endif
```

To avoid all of this repetition.

Reply:
@madolson sure, I will create a separate PR to address this repetition issue.